config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -208,14 +208,11 @@ periodics:
     preset-capz-windows-ci-entrypoint-common-main: "true"
     preset-capz-windows-azuredisk: "true"
   extra_refs:
-  - org: kubernetes
```

Comment:
we need this for https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/0187833c20b134404ec85b2280e7fd253effaba1/scripts/ci-entrypoint.sh#L78-L80

Reply:
This is a periodic job, so the "build k8s" flow will not happen.

Additionally, this job is totally broken:

https://testgrid.k8s.io/sig-windows-master-release#capz-azuredisk-windows-2019

cc @marosset

Reply:
Yea... we have a fix for that job in this PR :)
https://github.com/kubernetes/test-infra/pull/26397/files#diff-df283afbd02f2f70d68a1151d0b93b86f3ce1a8bf4766ef4d812aae58ac26a31R215

Reply:
Should I open a separate PR that sets workdir to the capz repo for that job since this PR his growing in scope?

Reply:
I'm fine with keeping it here, I think we just need to get consensus on these changes and merge this ASAP

Reply:
I added this repo checkout due to an error we saw with it not checkouted out: https://github.com/kubernetes/test-infra/pull/26358  but that line of code seems to have changed since then with https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/9cff770b5aa4de29fb7ec2d12bcd34bbd2f2d3b3

I guess I don't know the current state of what is needed for running this job against the master branch ok k/k using ci artifacts

Reply:
See my comment above: https://github.com/kubernetes/test-infra/pull/26397/files#r882907900

If this is meant to be a periodic job that runs against k/k HEAD I think setting `KUBERNETES_VERSION` to `"latest"` will do the trick.

Reply:
set to "latest"